README.md

```diff
@@ -1,15 +1,199 @@
+*YaraRulesApi* | [**yara_rules_delete_tag**](docs/YaraRulesApi.md#yara_rules_delete_tag) | **DELETE** /v1/environments/{environment}/yaraRules/{yaraRule}/tags/{tag} | DeleteTag
+*YaraRulesApi* | [**yara_rules_delete_yara_rule**](docs/YaraRulesApi.md#yara_rules_delete_yara_rule) | **DELETE** /v1/environments/{environment}/yaraRules/{yaraRule} | DeleteYaraRule
+*YaraRulesApi* | [**yara_rules_get_tag**](docs/YaraRulesApi.md#yara_rules_get_tag) | **GET** /v1/environments/{environment}/yaraRules/{yaraRule}/tags/{tag} | GetTag
+*YaraRulesApi* | [**yara_rules_get_yara_rule**](docs/YaraRulesApi.md#yara_rules_get_yara_rule) | **GET** /v1/environments/{environment}/yaraRules/{yaraRule} | GetYaraRule
+*YaraRulesApi* | [**yara_rules_list_tags**](docs/YaraRulesApi.md#yara_rules_list_tags) | **GET** /v1/environments/{environment}/yaraRules/{yaraRule}/tags | ListTags
+*YaraRulesApi* | [**yara_rules_list_yara_rules**](docs/YaraRulesApi.md#yara_rules_list_yara_rules) | **GET** /v1/environments/{environment}/yaraRules | ListYaraRules
+*YaraRulesApi* | [**yara_rules_update_yara_rule**](docs/YaraRulesApi.md#yara_rules_update_yara_rule) | **PATCH** /v1/environments/{environment}/yaraRules/{yaraRule} | UpdateYaraRule
+
+
+## Documentation For Models
+
+ - [Asset](docs/Asset.md)
+ - [Comment](docs/Comment.md)
+ - [DNSLookupResult](docs/DNSLookupResult.md)
+ - [Detection](docs/Detection.md)
+ - [FileAction](docs/FileAction.md)
+ - [HostnameMetadata](docs/HostnameMetadata.md)
+ - [IpAddressMetadata](docs/IpAddressMetadata.md)
+ - [ListAssetsResponse](docs/ListAssetsResponse.md)
+ - [ListCommentsResponse](docs/ListCommentsResponse.md)
+ - [ListObjectMetadataResponse](docs/ListObjectMetadataResponse.md)
+ - [ListObjectSightingsResponse](docs/ListObjectSightingsResponse.md)
+ - [ListObjectVariantsResponse](docs/ListObjectVariantsResponse.md)
+ - [ListOpinionsResponse](docs/ListOpinionsResponse.md)
+ - [ListTagsResponse](docs/ListTagsResponse.md)
+ - [ListYaraRulesResponse](docs/ListYaraRulesResponse.md)
+ - [MalEval](docs/MalEval.md)
+ - [MitreAttackTTP](docs/MitreAttackTTP.md)
+ - [NetworkIndicators](docs/NetworkIndicators.md)
+ - [ObjectDetonation](docs/ObjectDetonation.md)
+ - [ObjectMetadata](docs/ObjectMetadata.md)
+ - [ObjectSighting](docs/ObjectSighting.md)
+ - [ObjectSignature](docs/ObjectSignature.md)
+ - [ObjectVariant](docs/ObjectVariant.md)
+ - [Opinion](docs/Opinion.md)
+ - [RegistryKeyAction](docs/RegistryKeyAction.md)
+ - [Tag](docs/Tag.md)
+ - [TriggerObjectDetonationRequest](docs/TriggerObjectDetonationRequest.md)
+ - [X509Certificate](docs/X509Certificate.md)
+ - [YaraRule](docs/YaraRule.md)
+
+
+<a id="documentation-for-authorization"></a>
+## Documentation For Authorization
+
+
+Authentication schemes defined for the API:
+<a id="AuthToken"></a>
+### AuthToken
+
+- **Type**: API key
+- **API key parameter name**: Authorization
+- **Location**: HTTP header
+
+
+## Author
```

Comment:
should we remove this? (if it'll keep getting autogenerated, then we can keep as-is)